master/getting-started/kubernetes/installation/flannel.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Installing Calico on Kubernetes with flannel networking
+canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation/flannel'
+---
+
+## Before you begin
+
+- Ensure that you have a Kubernetes cluster that meets the 
+  {{site.prodname}} [system requirements](../../requirements). If you don't, 
+  follow the steps in [Using kubeadm to create a cluster](http://kubernetes.io/docs/getting-started-guides/kubeadm/) to create one.
+
+- If your cluster has RBAC enabled, issue the following command to 
```

Comment:
This feels weird in "Before you begin" - this is actually a part of installing Canal, so should be part of the installation step.